### PR TITLE
added errors to manager

### PR DESCRIPTION
**Pull request by banafshehderayat**

### CODE REVIEW

File: manager.php [hunk 1]
```diff
@@ -136,8 +163,12 @@
 								OCIBindByName($stid, ':bind2', $bind2);
 								OCIBindByName($stid, ':bind3', $bind3);
 								OCIBindByName($stid, ':bind4', $bind4);
-								OCIExecute($stid);
-								OCICommit($db_conn);	
+								$r = OCIExecute($stid);
+								if ($r == FALSE){
+									echo '<script type="text/javascript">'. 'alert("the employee information you entered is invalid")</script>';
```
Comment (grace-liang) [inline manager.php]: Can we make this upper case and have a period at the end?

File: manager.php [hunk 2]
```diff
@@ -91,21 +110,29 @@
...
                 					OCIBindByName($stid, ':bind2', $bind2);
 							OCIBindByName($stid, ':bind3', $bind3);
-               						OCIExecute($stid);
-							echo "Table Updated: " . "employee: " . $bind1 . " has been updated!";
-                					OCICommit($db_conn);
+               						$r = OCIExecute($stid);
+							if ($r == FALSE){
+
+								echo '<script type="text/javascript">'. 'alert("the employee information you entered is incorrect")</script>';
+							} else{
```
Comment (grace-liang) [inline manager.php:127]: Same here, capital and period?

File: manager.php [hunk 3]
```diff
@@ -77,7 +91,12 @@
                 			$stid = oci_parse($db_conn, $statement);
                 			$bind = $_POST['eID'];
                				OCIBindByName($stid, ':bind', $bind);
-                			OCIExecute($stid);
+                			$r = OCIExecute($stid);
+					if ($r == FALSE){
+
+								echo '<script type="text/javascript">'. 'alert("the employee ID you entered is incorrect")</script>';
+							} else {
```
Comment (grace-liang) [inline manager.php:94]: Same here. Add capital and period?

File: manager.php [hunk 4]
```diff
@@ -18,12 +18,26 @@
...
 	</div>
 	<div style="border: 1px solid black;">
-		Employee ID: <input type="text" name="eID">
-        	<input type="submit" value="Find Employee" name="findEmployee"></p>		
+		Employee ID:<select name="eID">
+		  		<?php 
+		  			require_once 'util.php';
+		  			$util2 = new Util;
+					$debug = True;
+					if ($debug) {
```
Comment (jasdeepk) [inline manager.php]: We should delete all these empty if cases

**Comment by jasdeepk** [comment]
:+1:

**Comment by jasdeepk** [comment]
Closing the pr but leaving the branch open in case we need to come back to it

**Pull request closed without merging**